emukit/core/acquisition/integrated_acquisition.py

```diff
@@ -0,0 +1,63 @@
+from typing import Union, Callable, Tuple
+
+import numpy as np
+
+from emukit.core.acquisition import Acquisition
+from emukit.core.interfaces import IModel, IPriorHyperparameters, IDifferentiable
+
+
+class IntegratedAcquisition(Acquisition):
+    """
+    This acquisition class provides functionality for integrating any acquisition function over model hyper-parameters
+    """
+    def __init__(self, model: Union[IModel, IPriorHyperparameters], acquisition_generator: Callable, n_samples: int=10):
+        """
+        :param model: An emukit model that implements IPriorHyperparameters
+        :param acquisition_generator: Function that returns acquisition object when given the model as the only argument
+        :param n_samples: Number of hyper-parameter samples to integrate over
```

Comment:
delete "to integrate over". Since we integrate over the hyper-parameters, not its samples.

Reply:
Done